src/main/java/com/github/splendor_mobile_game/game/model/Game.java

```diff
@@ -63,6 +63,17 @@ private void start(int playerCount) {
         // takeNobleTest();
     }
 
+    public int getTokenCount(TokenType type) {
+        return tokensOnTable.get(type);
+    }
+
+    public void changeTokens(Map<TokenType, Integer> tokenMap) {
```

Comment:
Add java doc to this function, because just looking at it it's not obvious what it does.